app/src/main/res/values/strings.xml

```diff
@@ -688,5 +688,6 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_generalFee_title">Does it cost a fee to enter %s?</string>
     <string name="quest_laundrySelfService_title">Is this laundry a self service laundry?</string>
     <string name="quest_handrail_title">Do these steps have a handrail?</string>
+    <string name="resurvey_opening_hours_title">"What are the opening hours of %s? Correct them if necessary."</string>
```

Comment:
UX wise, it is a bit odd to have an "OK" button for this question that is visible from the start on even so the user did not input anything yet. It could be misunderstood that for example pressing the "OK" button will lead to a form where one correct them or something.

More clear would be if there was a question like _"Are these opening hours for %s still correct?"_ and if the user clicks on "No", he is prompted to correct them. Technically it should be easily possible, make the opening hours unclickable first and on pressing no, exchange the yes-no button bar layout and make the opening hours form clickable.

This approach also solves another problem and that is how to treat opening hours that can not be displayed in the form: `24/7` at least. So, in the initial form, "24/7" (or "always open") can be printed in big letters in place of the opening hours input and only when the user clicks no, the normal opening hours form is displayed

Reply:
I mostly upgraded UX to this version. For now remaining things is `24/7` support and:

> make the opening hours unclickable first

Is it about avoiding accidental clicks resulting in confusion? It is not obvious that entries are clickable, but I think that it is OK to let user edit them also at this screen.

Reply:
Well, yes, the user would not expect the form to be editable at this point.
Easiest way to implement this is to put a transparent clickable view on top of the recycler view. Not sure if setting the recycler view to disabled would do anything.

Reply:
> Easiest way to implement this is to put a transparent clickable view on top of the recycler view.

Seems that recycler adapter needs to be edited anyway to get rid of delete buttons.